docs/usage/modules/platform.md

```diff
@@ -1,8 +1,8 @@
 # Renovate Platforms
 
-Renovate aims to be as platform-neutral as possible in general, while also taking advantage of any platform-specific features that are of benefit to users.
+Renovate aims to be platform-neutral, while also taking advantage of good platform-specific features.
 
-Currently supported platforms are:
+The currently supported platforms are:
 
 - Azure DevOps
```

Comment:
We should mention cloud and self-hosted support here

Reply:
```suggestion
- Azure DevOps (cloud, self-hosted)
```

Do you mean like this? I noticed we have Azure in the selfhosted docs: https://docs.renovatebot.com/self-hosting/#azure-devops

It seems that Azure DevOps can be run as a cloud instance or as a self-hosted instance.

Reply:
yes, for the other platforms we explicit mention cloud and self-hosted support, so it's a good idea to do that for azure devops too 🙃

Reply:
Do we want a link to the Azure DevOps homepage then?

Reply:
The on-premises version is called "Azure DevOps Server", and the cloud version is called "Azure DevOps Services" (though often just shortened to "Azure DevOps").

I know, the naming around these is a little confusing...

Reply:
Ok, so we should use two seperate entries, like we do with bitbucket

Reply:
I've made a new commit that splits it up into Azure DevOps and Azure DevOps Server.

Let me know if the new commit resolves this conversation. :wink:

Reply:
One small nit-pick: The URL for Azure DevOps is `dev.azure.com`, not `azure.microsoft.com`. Otherwise LGTM.

Reply:
> One small nit-pick: The URL for Azure DevOps is `dev.azure.com`, not `azure.microsoft.com`. Otherwise LGTM.

I'll go change that now then.